src/styles/globals.scss

```diff
@@ -0,0 +1,66 @@
+html,
+body {
+  padding: 0;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.lbh-container {
+  max-width: 1100px !important;
+}
+
+.lbh-form-group {
+  margin-bottom: 20px !important;
+}
+
+.lbu-service-switcher {
```

Comment:
Do you think these component specific styles should be pulled into `*.module.scss` files? Conscious that this global styles file might get huge

Reply:
yes - i was waiting to hear what we wanted to do with the react lib first. this is copied straight from my prototypes right now